src/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import "../styles/app.css";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+
+const App = () => {
+  const [results, setResults] = useState([]);
+  return (
+    <div className="app">
+      <header>
+        <img
+          className="nasa-logo"
+          src="https://upload.wikimedia.org/wikipedia/commons/a/a3/NASA_Worm_logo.svg"
```

Comment:
While this approach works, it would be perfectly reasonable to download the image and include as part of the repo as its not very large, and would prevent errors if the image was not available for some reason outside of your control